03-Login-app-directory/components/LogoutButton.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+import { useRouter } from "next/navigation";
+import { FC, MouseEventHandler } from "react";
+
+interface LogoutButtonProps {}
+
+const LogoutButton: FC<LogoutButtonProps> = ({}) => {
+  const router = useRouter();
+  const handleLogout: MouseEventHandler<HTMLDivElement> = () => {
+    localStorage.clear();
```

Comment:
In our [Passage.js](https://docs.passage.id/frontend/passage.js/session-reference) library, we now have a session class to help manage the user's session.